xray/core/indexing.py

```diff
@@ -117,25 +117,24 @@ def _try_get_item(x):
         return x
 
 
-def _get_loc(index, label, method=None):
-    """Backwards compatible wrapper for Index.get_loc, which only added the
-    method argument in pandas 0.16
-    """
-    if method is not None:
-        return index.get_loc(label, method=method)
-    else:
-        return index.get_loc(label)
-
-
-def convert_label_indexer(index, label, index_name='', method=None):
+def convert_label_indexer(index, label, index_name='', method=None,
+                          tolerance=None):
     """Given a pandas.Index (or xray.Coordinate) and labels (e.g., from
     __getitem__) for one dimension, return an indexer suitable for indexing an
     ndarray along that dimension
     """
+    # backwards compatibility for pandas<0.16 (method) or pandas<0.17
```

Comment:
What error do we get if we use the `tolerance` keyword with pandas 0.16?  Do we need to catch that ourselves?